server/@types/forms/index.d.ts

```diff
@@ -127,10 +131,6 @@ declare module 'inductionForms' {
     wantToAddQualifications: YesNoValue
   }
 
-  export interface HighestLevelOfEducationForm {
```

Comment:
HighestLevelOfEducationForm is no longer an "induction" form, it is just a form that can be used before (or as part of) an Induction. Hence moving it